src/modules/rc_update/rc_update.h

```diff
@@ -92,6 +92,8 @@ class RCUpdate : public ModuleBase<RCUpdate>, public ModuleParams, public px4::W
 
 private:
 
+	static constexpr uint64_t VALID_DATA_MIN_INTERVAL_US{1_s / 10}; // assume valid RC input is at least 10 Hz
```

Comment:
If 10Hz is the minimum rate then 1/10s would be the maximum interval no?
```suggestion
	static constexpr uint64_t VALID_DATA_MAX_INTERVAL_US{1_s / 10}; // assume valid RC input is at least 10 Hz
```

Reply:
> it can at the edge of reception as low as 3-4Hz

How about 5 Hz then?

Reply:
If we want to allow that use case I'd go for 4Hz because of 
![image](https://user-images.githubusercontent.com/4668506/161991656-42a00aef-a5ef-4a86-b3a2-d25581a2385c.png)
from https://www.team-blacksheep.com/tbs-crossfire-manual.pdf but I have no hard feelings. I would personally not really use this on purpose if not necessary. And I have no hard feelings because I agree that 10Hz is definitely the lowest limit for useful stick input in normal situations.